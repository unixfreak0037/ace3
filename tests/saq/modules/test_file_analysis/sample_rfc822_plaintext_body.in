***This is an external email message. Confirm the sender before opening attachments or clicking links. Avoid content that looks suspicious.***
________________________________
197.210.28.107

https://www.alienvault.com/
http://tldp.org/LDP/abs/html/

--
John W. Davison
unixfreak0037@gmail.com<mailto:unixfreak0037@gmail.com>