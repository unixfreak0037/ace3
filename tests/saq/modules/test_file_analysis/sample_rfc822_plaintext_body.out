https://www.alienvault.com
http://tldp.org/LDP/abs/html
